internal/commands/push/errors.go

```diff
@@ -1,10 +1,19 @@
 package push
 
-type errProjectNotFound struct {
+import (
+	"fmt"
+)
+
+type errProjectInvalid struct {
+	Path   string // app project path
```

Comment:
do these fields actually need to be `Exported`?

i'd also say we can remove this particular comment (considering its possible its _just a path_, and not anything app-related).  the comment on `Exists` is fine, or the field could be named `pathExists`